mods/alt-tab-per-monitor.wh.cpp

```diff
@@ -0,0 +1,227 @@
+// ==WindhawkMod==
+// @id              alt-tab-per-monitor
+// @name            Alt+Tab per monitor
+// @description     Pressing Alt+Tab shows all open windows on the primary display. This mod shows only the windows on the monitor where the cursor is.
+// @version         1.0.0
+// @author          L3r0y
+// @github          https://github.com/L3r0yThingz
+// @include         explorer.exe
+// @architecture    x86-64
```

Comment:
For 1.4 compatibility, try:
```suggestion
// @architecture    x86-64
// @compilerOptions -lole32 -loleaut32
```